index.html

```diff
@@ -3,9 +3,18 @@
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
 
+    <div class="container">
+        <div class="item item1">1</div>
```

Comment:
In de uitwerking van NOVI schrijven ze in het html bestand explicit voor iedere box "id":
<div class="item" id="item-a">A</div>
Ik zie dat jij dit niet hebt gedaan wat voor de uitwerking niet uitmaakt maar misschien voor een peer makkelijker leesbaar.